test/boostTest.cpp

```diff
@@ -110,21 +110,6 @@ int main( int argc, char* argv[] )
 	try
 	{
 		framework::init(fake_init_func, argc, argv);
-#if BOOST_VERSION >= 106000
```

Comment:
Ask @winsvega , the very first usage of boost internal code is by him. I just had to go in and add a new condition each time boost upgraded its minor version.